fastlane_core/lib/fastlane_core/feature/feature.rb

```diff
@@ -0,0 +1,11 @@
+module FastlaneCore
+  class Feature
+    attr_accessor :key, :description, :env_var, :experiment
```

Comment:
Consider making these `attr_reader`s, as having immutable value objects is always nice!